### ISSUE TITLE
Uppercase Dark for the name for NixOS

### ISSUE THREAD

**Issue by gelerum** [catppuccin/gtk#117]
You don't need to do all this, you just have the theme name incorrect. Lowercase d for dark.

_Originally posted by @eclairevoyant in https://github.com/catppuccin/gtk/issues/110#issuecomment-1676220875_
            

I was following README.MD and wasn't able to use the theme, but when i changed `d` to uppercase. The theme was applied. You should update README.MD

**Comment by eclairevoyant** [comment]
The correct name is lowercase d. You just have an out-of-date package, most likely. See #113.

**Comment by gelerum** [comment]
Yes, you are right. NixOS 23.05 use v4.1 of the theme. But it's the stable channel right now. Maybe write just a comment that 23.05 channel users should use uppercase d?
